src/utils/daos_hdlr.c

```diff
@@ -2806,16 +2973,15 @@
 			"Unable to allocate memory for destination path.");
 		D_GOTO(out, rc = -DER_NOMEM);
 	}
-	rc = fs_copy_parse_path(&dst_file_dfs, dst_str, dst_str_len,
-				&fa.dst_p_uuid, &fa.dst_c_uuid);
+	rc = dm_parse_path(&dst_file_dfs, dst_str, dst_str_len,
+		      	   &ca.dst_p_uuid, &ca.dst_c_uuid, daos_no_prefix);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs